TypeScript.YAML-tmLanguage

```diff
@@ -218,6 +218,7 @@ repository:
     - include: '#arrow-function'
     - include: '#paren-expression-possibly-arrow'
     - include: '#cast'
+    - include: '#nullish-coalesce'
```

Comment:
Instead of this you want to just add the matching rule to `expression-operators` and change ternary to see lookup next to `?` is not `?`